telegram/constants.py

```diff
@@ -1843,6 +1843,21 @@ class MessageType(StringEnum):
 
     .. versionadded:: 20.8
     """
+    REPLY_TO_STORY = "reply_to_story"
+    """:obj:`str`: Messages with :attr:`telegram.Message.reply_to_story`.
+
+    .. versionadded:: NEXT.VERSION
+    """
+    BOOST_ADDED = "boost_added"
+    """:obj:`str`: Messages with :attr:`telegram.Message.boost_added`.
+
+    .. versionadded:: NEXT.VERSION
+    """
+    SENDER_BOOST_COUNT = "sender_boost_count"
+    """:obj:`str`: Messages with :attr:`telegram.Message.sender_boost_count`.
+
+    .. versionadded:: NEXT.VERSION
+    """
```

Comment:
```suggestion
```
This is only an integer, and thus we don't need this

Reply:
I guess the important argument is that `sender_boost_count` can never be the sole attribute present but there always is some kind of other content in the message, right?

Reply:
yes. and test would fail without it

Reply:
hmm, I suppose I misunderstood what `constants.MessageType` is for